.pylintrc

```diff
@@ -16,7 +16,7 @@ clear-cache-post-run=no
 # In error mode, messages with a category besides ERROR or FATAL are
 # suppressed, and no reports are done by default. Error mode is compatible with
 # disabling specific errors.
-errors-only=
+#errors-only=
```

Comment:
Let's maybe take this out, since you added `--errors-only`

Reply:
That is the convention of pylintrc file - by default it was explained and commented out, similar to other config settings